src/lib/guesstimator/formatter/formatters/lib.js

```diff
@@ -40,7 +40,7 @@ export const normalTextMixin = Object.assign(
   {}, textMixin,
   {
     errors(g) { return this._normalTextErrors(g.text) },
-    guesstimateType(g) {
+    guesstimateType(g, defaultValue) {
```

Comment:
Why not put the line, ``low > 0 ? 'LOGNORMAL' : 'NORMAL'`` here?  This seems like the area where we select the guesstimateType.